frontend/src/app/modules/home/organization/create-organization/create-organization.component.sass

```diff
@@ -0,0 +1,29 @@
+::ng-deep
+    .new-org
+        width: 400px
+        min-width: 400px
+        max-width: 90vw
+        .p-dialog-header
+            padding: 1rem !important
+            border: 1px solid #dee2e6
+            padding: 1rem
+            background: #f8f9fa
```

Comment:
This "background" should be in variables, please correct it

Reply:
Done